pom.xml

```diff
@@ -1030,7 +1030,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
-            <version>2.9.1</version>
+            <version>2.9.2-SNAPSHOT</version>
```

Comment:
Revert this change

Reply:
good catch

Reply:
Sebastian merged the PR by mistake and addressed this issue in another PR : https://github.com/cdapio/hydrator-plugins/pull/1693